CardPayments/src/main/java/com/paypal/android/cardpayments/CardRequestFactory.kt

```diff
@@ -1,54 +1,76 @@
 package com.paypal.android.cardpayments
 
+import com.paypal.android.cardpayments.api.ApplicationContext
+import com.paypal.android.cardpayments.api.BillingAddress
+import com.paypal.android.cardpayments.api.CardAttributes
+import com.paypal.android.cardpayments.api.CardPaymentSource
+import com.paypal.android.cardpayments.api.ConfirmPaymentSourceRequest
+import com.paypal.android.cardpayments.api.PaymentSource
+import com.paypal.android.cardpayments.api.Verification
+import com.paypal.android.cardpayments.api.VerificationMethod
+import com.paypal.android.cardpayments.threedsecure.SCA
 import com.paypal.android.corepayments.APIRequest
 import com.paypal.android.corepayments.HttpMethod
-import org.json.JSONObject
+import kotlinx.serialization.InternalSerializationApi
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
 
-internal class CardRequestFactory {
+internal class CardRequestFactory(
+    private val json: Json = Json
```

Comment:
I'm still on the fence about this one. I guess I'm not sure what it does?

Reply:
My only concern is it may be confusing to someone new to the codebase. I'd honestly prefer to follow the [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) principle now and remove it since we don't have an immediate need to configure JSON parsing.